src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -298,6 +298,20 @@ export class BuildGraphTableComponent implements OnChanges {
     if ('graph' in changes && this.graph) {
       this.sortedBuilds = sortBuilds(this.graph);
     }
+
+    if(changes.showAllDependencies && (typeof changes.showAllDependencies.previousValue !== 'undefined') && (changes.showAllDependencies.previousValue != changes.showAllDependencies.currentValue))
```

Comment:
>(typeof changes.showAllDependencies.previousValue !== 'undefined') [](start = 38, length = 66)

Doesn't the next check implicitly check this? How can "undefined" be the current value? :-) Can simplify this check a bit

Reply:
Simplified!